client/src/pages/CalendarPage/CalendarPage.js

```diff
@@ -57,6 +57,8 @@ const CalendarPage = () => {
         return task.isStarted && "taskStartedAt" in task;
       case filterSet[2]:
         return !task.isStarted && "taskEndedAt" in task;
+      default:
```

Comment:
We can remove this default case since we already have the ternary check (if the filter was for "all")
![image](https://user-images.githubusercontent.com/24628243/164945846-e675bc74-b47f-4aef-ac86-9ce1d8069013.png)

Reply:
Removed.